packages/service-library/src/servicelib/archiving_utils.py

```diff
@@ -260,6 +264,10 @@ async def unarchive_dir(
                 f"Failed unarchiving {archive_to_extract} -> {destination_folder} due to {type(err)}."
                 f"Details: {err}"
             )
+            # maybe write unsupported error format here instead of all this to be able to still raise in case of error
+            if isinstance(err, NotImplementedError):
```

Comment:
I understand that the third-paryt library is the one raising NotImplementedError.
Can you add a link to the doc/code that supports this ? 
Perhaps a small test would not harm to make sure the assumption is valid across upgrades of the library.